pkg/relayer/proxy/errors.go

```diff
@@ -3,9 +3,11 @@ package proxy
 import sdkerrors "cosmossdk.io/errors"
 
 var (
-	codespace                       = "relayer/proxy"
-	ErrUnsupportedRPCType           = sdkerrors.Register(codespace, 1, "unsupported rpc type")
-	ErrInvalidRelayRequestSignature = sdkerrors.Register(codespace, 2, "invalid relay request signature")
-	ErrInvalidSession               = sdkerrors.Register(codespace, 3, "invalid session")
-	ErrInvalidSupplier              = sdkerrors.Register(codespace, 4, "invalid supplier")
+	codespace                            = "relayer/proxy"
+	ErrUnsupportedRPCType                = sdkerrors.Register(codespace, 1, "unsupported rpc type")
+	ErrInvalidRelayRequestSignature      = sdkerrors.Register(codespace, 2, "invalid relay request signature")
+	ErrInvalidSession                    = sdkerrors.Register(codespace, 3, "invalid session")
+	ErrInvalidSupplier                   = sdkerrors.Register(codespace, 4, "invalid supplier")
+	ErrUndefinedSigningKeyName           = sdkerrors.Register(codespace, 5, "undefined signing key name")
+	ErrUndefinedProxiedServicesEndpoints = sdkerrors.Register(codespace, 6, "undefined proxied services endpoints")
```

Comment:
@Olshansk similar question about convention regarding the error names. You and I had discussed it a bit in the on-chain context, I'm not sure if you feel like that should apply 1:1 here but wanted to call it out as it's an area where I wasn't sure if we had established / subscribed to a convention yet.

Reply:
Left a comment on this above